### ISSUE TITLE
Clean up control ids in itsg33.yaml

### ISSUE THREAD

**Issue by sleepycat** [PHACDataHub/zta-system-pattern#1]
The controls in the itsg33.yaml need the ids to be cleaned up.
Essentially the id of a control is a [composite key](https://en.wikipedia.org/wiki/Composite_key), that combines the family `AC`, a `-`, an id of 1-3 digits, in this case `2`, and an option enhancement number `(1)`.  Those elements should be represented separately, and then combined to create an id.
What we have:

```yml
id: AC2(1)
family: AC
class: Technical
title: Automated System Account Management
```
What we want:
```yml
control: AC-2(1)
id: 2
family: AC
enhancement: 1
class: Technical
title: Automated System Account Management
```

You'll need [`jq`](https://stedolan.github.io/jq/download/) and the yaml capable wrapper [`yq`](https://kislyuk.github.io/yq/) for this. Modify the `jq` [extract code](https://github.com/PHACDataHub/zta-system-pattern/blob/main/extractcontrols.jq) and then regenerate the yaml file with `yq --yaml-output -f extractcontrols.jq annotatedcontrols.json > itsg33.yaml`.

**Comment by LilaKelland** [comment]
Closed with PR #2

**Comment by LilaKelland** [comment]
PR #2 didn't fix issue correctly - removed any controls without an enhancement (i.e. AC-1)

**Comment by LilaKelland** [comment]
Closed with PR #3
